components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -115,8 +115,8 @@ module tomitribe_select {
         };
 
         function link(scope, element, attrs, uiSelect) {
-            scope.openOnFocusDelay = angular.isDefined(attrs.openOnFocusDelay) ? parseInt(attrs.openOnFocusDelay) : 0;
-            scope.openOnFocusDelayOnce = angular.isDefined(attrs.openOnFocusDelayOnce) ? attrs.openOnFocusDelayOnce : false;
+            scope.openOnFocusDelay = (angular.isDefined(attrs.openOnFocusDelay) && !_.isEmpty(attrs.openOnFocusDelay)) ? parseInt(attrs.openOnFocusDelay) : 0;
+            scope.openOnFocusDelayOnce = (angular.isDefined(attrs.openOnFocusDelayOnce) && !_.isEmpty(attrs.openOnFocusDelayOnce)) ? attrs.openOnFocusDelayOnce : false;
```

Comment:
what do you think about `angular.isDefined(attrs.openOnFocusDelayOnce) ? !!attrs.openOnFocusDelayOnce : false` here?

Reply:
Yep, you are right. Thanks